### DISCUSSION TITLE
About the paper model

### DISCUSSION THREAD

**Discussion by cyn2003** [neuralhydrology/neuralhydrology#210] [Q&A] [answered]
Hi，
![image](https://github.com/user-attachments/assets/5b555cad-62c0-45c8-9750-1a9cac415b2c)
I want to know whether the dynamic features and static features in the hindcast input are the same as those in the forecast input.
And I want to know what does the encoder LSTM output（green part model outputs）？
Could u please give me some details about the paper model？Thank u!

**Comment by gauchm** [comment, marked as answer]
> I want to know whether the dynamic features and static features in the hindcast input are the same as those in the forecast input.

Statics yes, dynamics no. The idea is that the dynamics in the forecast portion are actual forecast products with lead times. NeuralHydrology does not currently support lead times, but it does allow "simulating" forecasts via `forecast_inputs` (see https://neuralhydrology.readthedocs.io/en/latest/usage/config.html).

> And I want to know what does the encoder LSTM output（green part model outputs）？

The green boxes are the streamflow nowcast and forecasts. The first green box (output by the first LSTM) is the nowcast ("today's prediction for today"), the subsequent green boxes are the next days' forecasts. More precisely, each green box outputs the parameters of a CMAL distribution, see https://hess.copernicus.org/articles/26/1673/2022/.
